docs/intellicode-visual-studio-code.md

```diff
@@ -16,24 +16,43 @@ The [IntelliCode extension for Visual Studio Code](https://marketplace.visualstu
 
 ![IntelliCode for Python in Visual Studio Code](media/python-intellicode.gif)
 
-## Supported languages
-
 The supported languages for AI-assisted IntelliSense completions are:
 
 - Java
 - JavaScript and TypeScript
 - Python
 - SQL
 
-## Prerequisites
 
-To install the extension, you need Visual Studio Code October 2018 Release 1.29 or later.
+### Prerequisites
+
+The extension is supported by Visual Studio Code October 2018 Release 1.29 or later.
+
+AI-assisted IntelliSense for Java requires Java 8 Update 151 or higher. 
+## Whole-line autocomplete preview
```

Comment:
The way the page weaves between talking about the WLC extension and the main extension is confusing. The WLC extension also has a higher minimum version and narrower language support, which should be mentioned.

Reply:
+1 to Viv's comment here - we should clearly state the constraints/requirements for "whole line completion preview". 
We also need to ensure that if we're adopting "Whole-line autocomplete" we do so everywhere it is mentioned (docs AND product)

Reply:
I'm wondering whether one way to do this is to open the section with this text to make the link between the preview-ness and the extension clear. 

"IntelliCode also has a preview release that allows you to try out whole line completions for Python and JavaScript/TypeScript files.

This preview is provided via a separate extension, which you can acquire here [url]. It requires version X.Y.Z of Visual Studio Code or higher"

An even clearer way might be to move the whole section to its own file and link to it from here, making the separation cleaner. Once the preview period is over we could fold back here.